src/__tests__/Image/imagesReducer.test.js

```diff
@@ -153,7 +153,7 @@ describe('imagesReducer', () => {
         const action = {
           type: TYPES.UPDATE_IMAGE_LIKED,
           payload: {
```

Comment:
you could just test that it updates the image in state when the id is the same, not need to test image.liked or anything else